build/steps/build-android.yml

```diff
@@ -86,8 +86,8 @@ jobs:
       - task: CopyFiles@2
         displayName: 'Copy ${{ parameters.name }}'
         inputs:
-          SourceFolder: ${{ parameters.androidPath }}/bin/Debug/
-          Contents: '**/*.apk'
+          SourceFolder: ${{ parameters.androidPath }}/bin/Release/
+          Contents: '**/*AndroidControlGallery.apk'
```

Comment:
Is this change necessary?. With the previous code should take the .apk too, no?

Reply:
👍